src/Home/Home.jsx

```diff
@@ -1,72 +1,129 @@
-import React from 'react'
-import { QuoteData } from '../Data';
-import { useEffect,useState } from 'react';
-import "../Home/Home.css"
+import React from "react";
+import { QuoteData } from "../Data";
+import { useEffect, useState } from "react";
+import "../Home/Home.css";
+import axios from "axios";
 
 const Home = () => {
-    const randomQuote = Math.floor(Math.random() * Math.floor(QuoteData.length));
-
-    const [ques, setQues] = useState("what is your name?");
-    const [name, setName] = useState("");
-    const [userName, setUserName] = useState("");
-    const [goal, setGoal] = useState("");
-    const [userGoal, setUserGoal] = useState("");
-    const [nameStyle, setStyle] = useState();
-    const [goalStyle, setGoalStyle] = useState({ display: "none" });
-    const [userGoalStyle, setUserGoalStyle] = useState({ display: "block" });
-
-    function handelBtn() {
-        setQues("Good Evening");
-        setUserName(name);
-        setStyle({ display: "none" });
-        setGoalStyle({ display: "inline" });
-      }
-    
-      function handelGoal() {
-        setUserGoal(goal);
-        setUserGoalStyle({ display: "none" });
-      }
-
-    useEffect(() => {
-        document.body.style.backgroundImage =
-          "url('https://source.unsplash.com/1600x900/?')";
-      },[]); 
+  const randomQuote = Math.floor(Math.random() * Math.floor(QuoteData.length));
+  const newDate =  Date();
+
+  const [data, setData] = useState({});
+  const [ques, setQues] = useState("what is your name?");
+  const [name, setName] = useState("");
+  const [userName, setUserName] = useState("");
+  const [goal, setGoal] = useState("");
+  const [userGoal, setUserGoal] = useState("");
+  const [nameStyle, setStyle] = useState();
+  const [goalStyle, setGoalStyle] = useState({ display: "none" });
+  const [userGoalStyle, setUserGoalStyle] = useState({ display: "block" });
```

Comment:
Since you are using so many states I would suggest you to use a useReducer, but that is just a suggestion.